prime-angular-frontend/src/app/modules/adjudication/shared/components/site-registration-table/site-registration-table.component.html

```diff
@@ -52,6 +52,16 @@
             *matCellDef="let row;"> {{ row.siteDoingBusinessAs | default }} </td>
       </ng-container>
 
+      <ng-container matColumnDef="flagged">
```

Comment:
So it's not described the best in the ticket, but the flag icon should be placed in the status or state column like we do in the enrollee table. This just adds the icon after the text for the status. There should not be another column. see enrollee-table for a reference